### ISSUE TITLE
Updating README

### ISSUE THREAD

**Issue by RongBranovate** [jonhoo/vast-vmap#15]
Hi, first, i want to say job well done! :+1:  this lib is awesome! 

Thing is, the JSDocs are not all the clear to newbies.
Any chance you could please update the README with some info on how to approach different functionality? from using the queryVAST to Tracking and pixel handling etc... ?

Thanks in advance!

**Comment by jonhoo** [comment]
I agree with you that it would be good to also have a document describing how to get up and running. However, I never actually got around to using this library myself, as I switched jobs just as it was completed. Someone who is actively using might thus be better suited to write this than I. Patches welcome!
